aloe_webdriver/screenshot_failed.py

```diff
@@ -81,6 +102,9 @@ def take_screenshot(self):
     )
     base_name = re.sub(r'\W', '_', base_name, flags=re.UNICODE)
 
+    if SCREENSHOT_DIR:
```

Comment:
It should be possible to achieve this with `FORMAT` alone, just use `dirname` on `base_name`.